README.md

```diff
@@ -72,7 +75,10 @@ Example using save/load button and keeping the dialog open when pressing ok
 -----------------------------------------------------------------------------------------------------
     import argparse
     import sys
-    from PyQt4 import QtGui
+        try:
```

Comment:
git diff seems to show an indentation error here ?

Reply:
ha, quite right; it is a readme in my defence so I did it freehand!

Reply:
fixed.